ProjectSettings/EditorBuildSettings.asset

```diff
@@ -5,10 +5,19 @@ EditorBuildSettings:
   m_ObjectHideFlags: 0
   serializedVersion: 2
   m_Scenes:
+  - enabled: 0
+    path: 
+    guid: 00000000000000000000000000000000
+  - enabled: 1
+    path: Assets/Scenes/StartMenu.unity
+    guid: 25ce09e869240a84081359d05e6cc1b7
   - enabled: 1
     path: Assets/Scenes/LevelSelect.unity
-    guid: 947552f9a91cbca4bb4d6ff164212019
+    guid: 30277d48b45e13d4a8eb2a89feddf8cd
   - enabled: 1
-    path: Assets/Scenes/SampleScene.unity
+    path: Assets/Scenes/Level1.unity
```

Comment:
The code would be simpler if the level number corresponded directly to the level scene build index (meaning that the scene Level1 would have a build index of 1). I think it would be good to do that.

Reply:
How would we do that without forcing the build index of either main menu or level selection to change everytime we add a new level?